src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -16,7 +16,7 @@ namespace Altinn.Platform.Authentication.Controllers
     /// <summary>
     /// CRUD API for the System User 
     /// </summary>
-    //[Authorize]
+    [Authorize]
```

Comment:
End user need to also have the rights for the given party. So there needs to be a policy that protects that only users with system admin rights can do this. (we use same systemresource as access management)

Reply:
Yes, Simen is working on the SystemUserController